app/components/users.jsx

```diff
@@ -5,6 +5,7 @@ import { IntlMixin } from 'react-intl';
 
 import { replaceParams } from 'utils/localized-routes';
 
+/* eslint react/jsx-no-bind: 0 */
```

Comment:
Not really sure if this is the best way to handle this (and lang-picker.jsx) but I didn't really see a workaround in this scenario based on the [rules](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md)